infoschema/slow_log.go

```diff
@@ -319,11 +324,26 @@ func (st *slowQueryTuple) convertToDatumRow() []types.Datum {
 	} else {
 		record = append(record, types.NewIntDatum(0))
 	}
+	record = append(record, types.NewStringDatum(parsePlan(st.plan)))
 	record = append(record, types.NewStringDatum(st.prevStmt))
 	record = append(record, types.NewStringDatum(st.sql))
 	return record
 }
 
+func parsePlan(planString string) string {
+	if len(planString) <= len(variable.SlowLogPlanPrefix)+len(variable.SlowLogPlanSuffix) {
+		return planString
+	}
+	planString = planString[len(variable.SlowLogPlanPrefix) : len(planString)-len(variable.SlowLogPlanSuffix)]
+	decodePlanString, err := plancodec.DecodePlan(planString)
```

Comment:
The format is pretty unfriendly when plan is displayed as decoded in `information_schema.slow_query` table.

Reply:
```sql
mysql> select plan, time from information_schema.slow_query where plan != "" order by query_time desc limit 3\G
*************************** 1. row ***************************
plan:   Projection_6            root    10      Column#3, Column#4
        └─Sort_7                root    10      Column#3:asc, Column#5:asc
          └─Selection_10        root    10      eq(Column#2, "information_schema")
            └─MemTableScan_9    root    10
time: 2019-10-17 19:46:56.412449
*************************** 2. row ***************************
plan:   Projection_8            root    3                       Column#28, Column#1
        └─TopN_9                root    3                       Column#6:desc, offset:0, count:3
          └─Selection_16        root    6656.666666666667       ne(Column#28, "")
            └─MemTableScan_15   root    6656.666666666667
time: 2019-10-17 19:48:30.240318
*************************** 3. row ***************************
plan:   Projection_6            root    10      Column#3, Column#4
        └─Sort_7                root    10      Column#3:asc, Column#5:asc
          └─Selection_10        root    10      eq(Column#2, "information_schema")
            └─MemTableScan_9    root    10
time: 2019-10-17 19:47:13.902430
3 rows in set (0.00 sec)

mysql> select plan, time from information_schema.slow_query where plan != "" order by query_time desc limit 3;
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+
| plan                                                                                                                                                                                                                                                                        | time                       |
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+
|       Projection_6            root    10      Column#3, Column#4
        └─Sort_7                root    10      Column#3:asc, Column#5:asc
          └─Selection_10        root    10      eq(Column#2, "information_schema")
            └─MemTableScan_9    root    10                                                                         | 2019-10-17 19:46:56.412449 |
|       Projection_8            root    3                       Column#28, Column#1
        └─TopN_9                root    3                       Column#6:desc, offset:0, count:3
          └─Selection_16        root    6656.666666666667       ne(Column#28, "")
            └─MemTableScan_15   root    6656.666666666667                    | 2019-10-17 19:48:30.240318 |
|       Projection_6            root    10      Column#3, Column#4
        └─Sort_7                root    10      Column#3:asc, Column#5:asc
          └─Selection_10        root    10      eq(Column#2, "information_schema")
            └─MemTableScan_9    root    10                                                                         | 2019-10-17 19:47:13.902430 |
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------+
```